zone/inventory.cpp

```
@@ -650,10 +674,14 @@ bool Client::SummonItem(uint32 item_id, int16 charges, uint32 aug1, uint32 aug2,
 	}
 
 	// put item into inventory
+	bool item_persisted = false;
 	if (to_slot == EQ::invslot::slotCursor) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)